Adyen.EcommLibrary/HttpClient/HttpURLConnectionClient.cs

```diff
@@ -79,21 +71,23 @@ public async Task<string> RequestAsync(string endpoint, string json, Config conf
                 streamWriter.Close();
             }
 
-            using (var response = (HttpWebResponse) await httpWebRequest.GetResponseAsync())
+            try
             {
-                using (var reader = new StreamReader(response.GetResponseStream(), _encoding))
-                {
-                    responseTextTask = reader.ReadToEndAsync();
-                }
-
-                if (response.StatusCode != HttpStatusCode.OK)
+                using (var response = (HttpWebResponse)await httpWebRequest.GetResponseAsync())
                 {
-                    var httpClientException = new HttpClientException((int) response.StatusCode, "HTTP Exception",
-                        response.Headers, responseTextTask.Result);
-                    throw httpClientException;
+                    using (var reader = new StreamReader(response.GetResponseStream(), _encoding))
+                    {
+                        responseText = await reader.ReadToEndAsync();
+                    }
                 }
             }
-            return await responseTextTask;
+            catch (WebException e)
+            {
+                Console.WriteLine(e);
```

Comment:
To avoid printing on stdout, I believe is better to log this by using the client logger if it is set: https://github.com/Adyen/adyen-dotnet-api-library/blob/63eda4a1a379a2605d3f9247e128edd632013041/Adyen.EcommLibrary/Client.cs#L96

Reply:
True. It ended being included when I copied across from the non async version of Request to make them the same. I can update use Logline in this PR for both.

Reply:
Removed the try catch and logging as this should be done higher up in the abstract service or just leave it to the consumer of the library in my opinion. What do you think.

Reply:
Yes it is true. The try catch can be removed as the exceptions are handled in higher level